### PR TITLE
Brand the devdocs 404 page in PrestaShop spirit

### PR DESCRIPTION
**Pull request by matks**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Use Preston SVG for HTTP 404. The devdocs 404 looks like https://devdocs.prestashop.com/jajajaha today so it's not branded. I reused the SVG from https://www.prestashop.com/en/ajajaja . Also I remove the `¯\\_(ツ)_/¯` that I find not serious.
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes https://github.com/PrestaShop/docs/issues/870
| How to test?      | Builds the site, then visit a 404 page.
| Possible impacts? |

### CODE REVIEW

File: layouts/404.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>{{T "title-404"}}</h1>
           <p>{{T "message-404"}}</p>
           <p></p>
           <p><a href='{{ "" | relLangURL }}'>{{T "Go-to-homepage"}}</a></p>
-          <p><img src='{{ "/images/gopher-404.jpg" | relURL }}' style="width:50%" alt="Page not found!"></p>
+          <p><img src='{{ "/images/prestashop-404.svg" | relURL }}' style="width:50%" alt="Page not found!"></p>
```
Comment (NeOMakinG) [inline layouts/404.html:50]: Is the width 50% fine with preston? Looks like it's really large

Comment (matks) [reply layouts/404.html:50]: I like it like this, quite big, but yes we can make it better looking.

I'm a backend dev, I'm not qualified to say whether something looks fine or not ^^

Comment (NeOMakinG) [reply layouts/404.html:50]: If you like it like this, it's fine then, I didn't check it myself

The fact that you're not a front-end dev or a designer doesn't mean that you can't visually validate something don't you think? 🤣

Comment (PierreRambaud) [reply layouts/404.html:50]: I think it's ok to keep to like that for the moment and improve it if needed in the future :)

**Review by PierreRambaud — APPROVED** [review]

File: layouts/404.html (hunk 1 above)
Comment (matks) [reply layouts/404.html:50]: > The fact that you're not a front-end dev or a designer doesn't mean that you can't visually validate something don't you think? 🤣

What I call "fine" or "pretty" is sometimes not approved by frontend developers. I'm too tolerant it seems.

I find bootstrap good-looking for instance XD

**Comment by matks** [comment]
Thank you for review

**Comment by matks** [comment]
Mmmm 🤔 there is an issue with the image.
https://devdocs.prestashop.com/aaa

I guess some build step is missing.

I think the image should be in static/ https://github.com/PrestaShop/ps-docs-theme/tree/main/static but it is not.

**Pull request merged**